src/Sulu/Bundle/AdminBundle/Resources/js/components/Schedules/FixedSchedule.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import React from 'react';
+import {transformDateToDateTimeString} from '../../utils/Date';
+import {translate} from '../../utils/Translator';
+import DatePicker from '../DatePicker';
+import Form from '../Form';
+import type {FixedScheduleEntry} from './types';
+
+type Props = {|
+    index: number,
+    onChange: (index: number, value: FixedScheduleEntry) => void,
+    value: FixedScheduleEntry,
+|};
+
+export default class FixedSchedule extends React.Component<Props> {
+    handleStartChange = (start: ?Date) => {
+        const {index, onChange, value} = this.props;
+        onChange(index, {...value, start: transformDateToDateTimeString(start)});
+    };
+
+    handleEndChange = (end: ?Date) => {
+        const {index, onChange, value} = this.props;
+        onChange(index, {...value, end: transformDateToDateTimeString(end)});
+    };
+
+    render() {
+        const {value} = this.props;
+
+        return (
+            <Form>
+                <Form.Field colSpan={6} label={translate('sulu_admin.start')}>
+                    <DatePicker
```

Comment:
would use `00:00` as placeholder here and fallback to that value on the server if nothing is set

Reply:
That would not be the correct behavior... It should also be possible to omit start or end, and interpret it as an open interval then...

The only way I see making this work with validation is to use the `schema` node we have introduced for the [`page_settings` form](https://github.com/sulu/sulu/blob/ad20c49418e7cb2a0fd54b0656c527f34b0cc432/src/Sulu/Bundle/PageBundle/Resources/config/forms/page_settings.xml#L8-L49), but I don't think that I am able to finish that on time... This schema is to simple until now, because it does not support any arrays, which we would need because this is within a block.

I think the best bet is to leave it as it is, and improve the schema XML tag in a separate PR.

Reply:
you are right, i thought i added this comment to the `WeeklySchedule` 🙈